woocommerce/woocommerce-gateway-stripe
# Replace Sources API with Payment Methods API when listing saved credit cards and SEPA debits

# Changes proposed in this Pull Request:

Issue: #1502

This PR replaces the Sources API calls with PaymentMethods API when listing saved credit cards and SEPA debits from Stripe. There is no visible change for the merchant and/or the customer, it's just a behind the scenes upgrade.

> _Why?_
Stripe recommends using the PaymentsMethods API instead of sources and tokens because new PaymentMethods added to a customer will not be duplicated in the sources property of the customer object.

# Testing instructions
0. Make sure that both `Stripe` and `Stripe SEPA Direct Debit` payment methods are enabled.
1. Go to My Account / Payment methods.
2. Verify the currently saved payment methods.
3. Go to Cart / Checkout.
4. Select "Credit Card (Stripe)" or "SEPA Direct Debit".
4. Verify that all the saved cards (or SEPA IBAN are listed.
5. Complete the checkout with the selected saved payment method.

-------------------
